test/expected_loglik.jl

```diff
@@ -45,4 +45,20 @@
         GaussHermite(), zeros(10), q_f, GaussianLikelihood()
     ) isa Real
     @test ApproximateGPs._default_quadrature(θ -> Normal(0, θ)) isa GaussHermite
+
+    @testset "testing Zygote compatibility with GaussHermite" begin # see issue #82
+        N = 10
+        gh = GaussHermite(12)
+        μ = randn(rng, N)
+        σ = rand(rng, N)
+        for lik in likelihoods_to_test
+            y = rand.(rng, lik.(rand.(Normal.(μ, σ))))
+            g = only(
+                Zygote.gradient(μ) do x
+                    ApproximateGPs.expected_loglik(gh, y, Normal.(x, σ), lik)
```

Comment:
would it be helpful to also test gradients wrt \sigma? (and maybe parameters of likelihood?)

Reply:
So far I don't think any of the likelihoods have parameters but I can add tests for sigma

Reply:
Both GaussianLikelihood and GammaLikelihood do..

Reply:
Ah yeah true! I can add specific tests for those then